test/libsolidity/smtCheckerTests/external_calls/external_call_with_value_3.sol

```diff
@@ -0,0 +1,18 @@
+interface I {
+	function f() external payable;
+}
+
+contract C {
+	function g(I i) public {
+		require(address(this).balance > 100);
+		i.f{value: 20}();
```

Comment:
Maybe test with a call to `this.g{value: ...}()` would be useful too.

Reply:
Nice catch, that's somehow broken.